src/main.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import ReactDOM from 'react-dom/client'
+import App from './App'
+import './index.css'
+
+ReactDOM.createRoot(document.getElementById('root')).render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+)
```

Comment:
It is better if you remove React on this lines of codes.
```suggestion
  <StrictMode>
    <App />
  </StrictMode>
)
```